maths/sum_of_arithmetic_series.py

```diff
@@ -0,0 +1,12 @@
+# DarkCoder
+def sumOfSeries(a, d, n):
```

Comment:
Hi @TheRealDarkCoder! Please follow the snake case convention in Python. The function should be named `sum_of_series` rather than `sumOfSeries`.